src/charm.py

```diff
@@ -1424,6 +1456,10 @@ def update_config(self, is_creating_backup: bool = False) -> bool:
             logger.debug("Early exit update_config: Patroni not started yet")
             return False
 
+        # Try to connect
+        if not self._can_connect_to_postgresql:
+            logger.debug("Early exit update_config: Cannot connect to Postgresql")
+            return False
```

Comment:
If the node's db is not yet up, we can't really check those values. Next update status should run this again.

Reply:
Changed to warning.